src/uct/api/uct.h

```diff
@@ -363,14 +363,69 @@ enum uct_cb_flags {
  * @brief Mode in which to open the interface.
  */
 enum uct_iface_open_mode {
-   UCT_IFACE_OPEN_MODE_DEVICE          = UCS_BIT(0),  /**< Interface is opened on a specific device */
-   UCT_IFACE_OPEN_MODE_SOCKADDR_SERVER = UCS_BIT(1),  /**< Interface is opened on a specific address
-                                                           on the server side */
-   UCT_IFACE_OPEN_MODE_SOCKADDR_CLIENT = UCS_BIT(2)   /**< Interface is opened on a specific address
-                                                           on the client side */
+   /**< Interface is opened on a specific device */
+   UCT_IFACE_OPEN_MODE_DEVICE          = UCS_BIT(0),
+
+   /**< Interface is opened on a specific address on the server side. This mode
+        is going to be deprecated in near future for a better API. */
+   UCT_IFACE_OPEN_MODE_SOCKADDR_SERVER = UCS_BIT(1),
+
+   /**< Interface is opened on a specific address on the client side This mode
+        is going to be deprecated in near future for a better API. */
+   UCT_IFACE_OPEN_MODE_SOCKADDR_CLIENT = UCS_BIT(2)
```

Comment:
Remove the '<' marker in the comment blocks since they come before, rather than after, the enum member.